AutocompleteClient/Utils/String/String+SearchSuggestionsSection.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-extension String {
+public extension String {
```

Comment:
If we publicly extend string would that extension propagate to consumers?

Reply:
Yes.

Reply:
Is there another way to do this?  I'm not sure that consuming apps would love that we're extending common classes like string